### ISSUE TITLE
Ordered lists does not have numbers in several themes in FF

### ISSUE THREAD

**Issue by arnosthavelka** [hakimel/reveal.js#1339]
I just downloaded new version 3.1.0 and found out this issue in FF 40.0.2 (Chrome and IE are OK). It happens at least in themes beige, sky and serif.

You can see it here http://lab.hakim.se/reveal-js/#/15 (when you change theme appropriately).

**Comment by waywaaard** [comment]
It does also happen in white. It seems that the alpha-numeric symols disappear when you apply the `transform: scale(1.3)` on the slides wrapper. If you just translate but do not scale the numbers are shown.

**Comment by Delapouite** [comment]
Testing on Firefox 42.0 on Arch and I can't reproduce this weird display bug.

**Comment by waywaaard** [comment]
It seems to be caused by some custom CSS. It does not happen on reveal.js demo.

**Comment by waywaaard** [comment]
It is somehow related to the colors. If you have a theme with a light background and a blackish color, the numbers are not visible. However, if you change the text color to white and add a black background the numbers are shown. o.O

**Comment by waywaaard** [comment]
@hakimel ping

**Comment by hakimel** [comment]
Gave it a try but I'm not able to reproduce this in FF 42 / OS X. The numbers are always visible in the beige/sky themes, even when the presentation is scaled up.

**Comment by waywaaard** [comment]
Still happening in FF 42.0 Win 10

With css transform applied:
![image](https://cloud.githubusercontent.com/assets/1634468/11211878/d568f0fc-8d32-11e5-8d2b-f8abc0384f5c.png)

without or scale set to 1:
![image](https://cloud.githubusercontent.com/assets/1634468/11211905/f92449e2-8d32-11e5-8d37-e8f86df3e7e7.png)

**Comment by LWisteria** [comment]
This seems to be a bug of Firefox Win?

For emergency avoidance, edit [theme-what-you-want].css

```css
.reveal ol {
  counter-reset: ol;
  list-style-type: none; }
  
.reveal ol li:before {
  counter-increment: ol;
  content: counter(ol) ". "; }
```

**Comment by olberger** [comment]
Confirmed working in FF 45.3 on Linux, and not working in FF 48 on windows 7 :-/

**Comment by olberger** [comment]
@LWisteria Tried it and solves some cases but not when:
```
<ol>
  <li>
    <ul>
      <li>
```
where enclosed <ul> items will be numbered in the same sequence as enclosing, whereas they shouldn't.

**Comment by LWisteria** [comment]
I found this problem has been solved on Firefox 57.0 64bit Windows 10. How about you all?

**Comment by bnjmnt4n** [comment]
This seems like some weird rendering bug in older versions of Firefox for Windows. Closing for now since I can't replicate this anymore.
